### DISCUSSION TITLE
Create a conversation select modal element

### DISCUSSION THREAD

**Discussion by jloh** [slack-go/slack#988] [Q&A]
Hi all. I'm using this library to create a modal in response to a slash command. I'm currently trying to add a conversation select to the modal response outlined here https://api.slack.com/surfaces/modals/using#modal_response_url in the slack docs. Basically I need to create this in my response:

```
{
    "type": "modal",
    ...
    "blocks": [
      ...
      {
        "block_id": "my_block_id",
        "type": "input",
        "optional": true,
        "label": {
          "type": "plain_text",
          "text": "Select a channel to post the result on",
        },
        "element": {
          "action_id": "my_action_id",
          "type": "conversations_select",
          "response_url_enabled": true,
        },
      },
    ],
  };
}
```

I can see this is possible by the `SelectBlockElement` but I can't get it working. I get stuck using the `NewOptionsSelectBlockElement` func and I'm unsure how to piece it all together.

Any hints/direction on how to achieve the above would be fantastic. Thanks in advance!

**Comment by karl-johan-grahn** [comment]
Have you looked at https://github.com/slack-go/slack/blob/master/examples/modal_users/users.go?
